### DISCUSSION TITLE
Stash window doesn't respond and side colors even without stashed windows

### DISCUSSION THREAD

**Discussion by Solar-io** [rxhanson/RectanglePro-Community#525] [Q&A]
Hi, I love stashed windows and use them all day long. I have two issues, though. The first issue is that at some point in time, windows will no longer respond to my typical cmd+mouse at the screen edge. So, I have to drag them out from the side since they are still stashed and then re-stash them. It will only happen to one window at a time and not all of them simultaneously. My second issue is that I have one window stashed on the right side, but it displays three color bars as if three windows are stashed. I have checked multiple times, and there is just one window. Any suggestions would be helpful.

**Comment by rxhanson** [comment]
Thanks for reporting these to me. 

I can't reproduce either of these up front, so it's going to take some investigation to figure out what exactly is going on. 

At a start, enable the checkbox "Show in menu" in the Stash tab of the Settings window. For both of these scenarios, when they happen, go to the "Unstash" menu item in the menu bar menu and let me know what Rectangle Pro is stating as stashed windows. That should give a little insight as to where things might be going wrong.

**Comment by brucetm** [comment]
@rxhanson I looked and did not find a bug for this, so commenting here instead, please let me know if there is a better place to put this info:

Windows stashed by an app layout can get 'lost' by the app. That is, they ARE stashed, but after some time, i.e., a day or two of sleep wake cycles, they can fall out of the app's 'stashed' list and so not able to be unstashed anymore by gesture. Talk through of what I mean:  [Screen recording in Google Drive](https://drive.google.com/file/d/1AcJ-qQoN5z58gOQf28oS8l9IM2gDZv5C/view?usp=sharing)

**Reply by rxhanson** [reply]
Thanks! I've converted this into an issue so we can track it there: https://github.com/rxhanson/RectanglePro-Community/issues/609
